src/main/java/com/yourcodereview/jegors/task1/Main.java

```diff
@@ -38,21 +42,20 @@ public static void main(String[] args) {
         var startTime = Instant.now();
         LOGGER.log(INFO, "Execution start time: {0}", startTime);
 
-        process(path);
+        try (var ips = Files.lines(path)) {
```

Comment:
@rabestro 
Is it necessary to  create string object for each IP address? Maybe it could be read symbol by symbol  with composing IP integer on the way?